conanfile.py

```diff
@@ -232,6 +232,11 @@ def _disableQtOptions(disableFlag, options):
         if self.settings.os == "Android":
             self.options["sqlite3"].build_executable = False
 
+        # Needed for mobile builds (see https://github.com/vcmi/vcmi/pull/4602)
+        self.options["onetbb"].tbbmalloc = False
+        self.options["onetbb"].tbbproxy = False
+        self.options["onetbb"].tbbbind = False
```

Comment:
please place them before `self.options["sdl"]`

also the comment isn't needed as we don't use those parts anyway